tests/async/test_subscribe_async.py

```diff
@@ -0,0 +1,75 @@
+
+# vim:ts=4:sw=4:et:
+# Copyright 2015-present Facebook, Inc.
+# Licensed under the Apache License, Version 2.0
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+# no unicode literals
+
+import AsyncWatchmanTestCase
+import tempfile
+import os
+import os.path
+try:
+    import unittest2 as unittest
+except ImportError:
+    import unittest
+import pywatchman
+
+
+class TestSubscribe(AsyncWatchmanTestCase.AsyncWatchmanTestCase):
+
+    def test_subscribe(self):
```

Comment:
I still think a test for an exception case (mocked or a real empty response, I don't care) would be nice. You'd catch the `except ex:` NameError that way.